server/pages/index.js

```diff
@@ -40,6 +43,24 @@ const staticFiles = [
 
 const sections = sectionsModule.get();
 
+// Retrieve Promise for the contents of `filepath`. On error, resolve with
+// empty string.
+function getFileContentsSafely( filepath ) {
```

Comment:
@nb I added "safely" because I'm handling the error case and returning a default value. So callers of this function don't have to worry about errors.